queue/util/createConsumer.js

```diff
@@ -47,6 +47,8 @@ module.exports = async function (options) {
     } catch (err) {
       logger.error('Error handling message:', err)
 
+      // TODO: This needs more thought. If a message is continuously failing,
+      // we shouldn't keep requeuing
```

Comment:
We should allow the consumer on `windbreaker` handle that. Maybe provide special handler for rejected messages to give more flexiibility.